src/Documentation/Summarizer/summarize_documentation.py

```diff
@@ -53,6 +54,76 @@ def items_of_kind(items, kind):
         )
     ]
 
+def summaries_table(namespace, kind, header):
+    table_header = f"""
+## {header}
+
+| Name | Summary |
+|------|---------|
+"""
+    def first_line(summary):
+        lines = summary.splitlines()
+        return lines[0] if lines else ""
+
+    return (
+        table_header + "\n".join(
+            f"|[{item['name']}](xref:{item['uid']}) |{first_line(item['summary'])} |"
```

Comment:
Is there no space after the first `|` of each column?

Reply:
In the examples for formatting for docs.microsoft.com, non-heading cells did not have a space after the first `|`. It may well be that the examples are incorrect, but we'll need to check the reference rendering on docs.ms to check that.

Reply:
OK, I think it's fine without a space, just curious if there was a reason.